x-pack/plugins/canvas/public/components/sidebar_header/sidebar_header.tsx

```diff
@@ -0,0 +1,329 @@
+          </EuiToolTip>
+        </EuiFlexItem>
+      </Fragment>
+    );
+  };
+
+  private _getLayerMenuItems = () => {
+    const { bringToFront, bringForward, sendBackward, sendToBack } = this.props;
+
+    return {
+      name: 'Order',
+      className: topBorderClassName,
+      panel: {
+        id: 1,
+        title: 'Order',
+        items: [
+          {
+            name: 'Bring to front', // TODO: check against current element position and disable if already top layer
+            icon: 'sortUp',
+            onClick: () => bringToFront(),
+          },
+          {
+            name: 'Bring forward', // TODO: same as above
+            icon: 'arrowUp',
+            onClick: () => bringForward(),
+          },
+          {
+            name: 'Send backward', // TODO: check against current element position and disable if already bottom layer
+            icon: 'arrowDown',
+            onClick: () => sendBackward(),
+          },
+          {
+            name: 'Send to back', // TODO: same as above
+            icon: 'sortDown',
+            onClick: () => sendToBack(),
+          },
+        ],
+      },
+    };
+  };
+
+  // TODO: restore when group and ungroup can be triggered outside of workpad_page
+  // private _getGroupMenuItem = () => {
+  //   const { groupIsSelected, ungroupNodes, groupNodes } = this.props;
+  //   return groupIsSelected
+  //     ? {
+  //         name: 'Ungroup',
+  //         className: topBorderClassName,
+  //         onClick: () => {
+  //           ungroupNodes();
+  //         },
+  //       }
+  //     : {
+  //         name: 'Group',
+  //         className: topBorderClassName,
+  //         onClick: () => {
+  //           groupNodes();
+  //         },
+  //       };
+  // };
+
+  private _renderPanelTree = (closePopover: () => void) => {
+    const {
+      showLayerControls,
+      cutNodes,
+      copyNodes,
+      pasteNodes,
+      deleteNodes,
+      cloneNodes,
+    } = this.props;
+
+    const items = [
+      {
+        name: 'Cut',
+        icon: 'cut',
+        onClick: () => {
+          cutNodes();
+          closePopover();
+        },
+      },
+      {
+        name: 'Copy',
+        icon: 'copy',
+        onClick: () => copyNodes(),
+      },
+      {
+        name: 'Paste', // TODO: can this be disabled if clipboard is empty?
+        icon: 'copyClipboard',
+        onClick: () => {
+          pasteNodes();
+          closePopover();
+        },
+      },
+      {
+        name: 'Delete',
+        icon: 'trash',
+        onClick: () => {
+          deleteNodes();
+          closePopover();
+        },
+      },
+      {
+        name: 'Clone',
+        onClick: () => {
+          cloneNodes();
+          closePopover();
+        },
+      },
+      // TODO: restore when group and ungroup can be triggered outside of workpad_page
+      // this._getGroupMenuItem(),
+    ];
+
+    if (showLayerControls) {
+      // @ts-ignore - this is the right shape of an EUI panel tree
```

Comment:
If there is a shape for the EUI panel tree, import it and declare `items` to be a collection of them...

Reply:
also borrowed from `@elastic/eui` types to fix this error